instrumentation/opentelemetry-instrumentation-urllib/src/opentelemetry/instrumentation/urllib/__init__.py

```diff
@@ -297,17 +297,17 @@ def _create_client_histograms(meter) -> Dict[str, Histogram]:
         MetricInstruments.HTTP_CLIENT_DURATION: meter.create_histogram(
             name=MetricInstruments.HTTP_CLIENT_DURATION,
```

Comment:
Would a rename of the name be needed as well since metadata is name + description?

Reply:
The match is made on name, and the check is to ensure the description and data type conform to metrics already received with the same name. So only the description needs to be updated.